.importjs.js

```diff
@@ -1,36 +0,0 @@
-module.exports = {
```

Comment:
This was config for a development tool that only I was using, that I now don't use.